pkg/daemon/ceph/client/crush.go

```diff
@@ -201,6 +201,7 @@ func formatProperty(name, value string) string {
 
 // GetOSDOnHost returns the list of osds running on a given host
 func GetOSDOnHost(context *clusterd.Context, clusterInfo *ClusterInfo, node string) (string, error) {
+	node = strings.ReplaceAll(node, ".", "-")
```

Comment:
Is this the only difference between the node names and CRUSH names?

Reply:
Yes!
Nevertheless, I changed the code to use the already existing function to normalize the node name.